command-expansion/build.gradle

```diff
@@ -0,0 +1,25 @@
+
+plugins {
+  id "com.github.node-gradle.node" version "3.1.1"
+}
+
+node {
+  version = '16.0.0'
+  npmVersion = '8.5.0'
+  download = true
+}
+
+task build(type: NpxTask) {
+  dependsOn npmInstall
+
+  command = 'npx'
```

Comment:
I'm curious about the pros/cons of putting together the command here, vs using the npm run tasks directly.
(see [docs](https://github.com/node-gradle/gradle-node-plugin/blob/master/docs/usage.md#executing-npm-tasks)

For example, depending on the `npm_build` task would be equivalent to running `npm run build`, which is defined to be `npx tsc` in package.json.

One possible advantage of your way is that specifying the input files could prevent unnecessary re-builds? But I'm just guessing here.

Reply:
I wanted to make things simpler and consistent by using the same command `./gradlew build` which will build all the other Aerie services. You can `./gradlew npm_build` too, but I felt that was one extra thing to remember so I defined a `build` task instead.

Reply:
Good point :+1: I like the consistency as well.

I'm noticing that the first line is `dependsOn npmInstall`. Would it still achieve what we're trying to do if we added another line `dependsOn npm_build`?

I'm not super familiar with gradle, so I'm not sure this is the "right way" to call one task from another task. I'm curious to hear your thoughts.

Reply:
Thanks for pointing this out. I reworked the task into npm type which is a lot cleaner